Gravity/Gravity/Base/BaseDto.cs

```diff
@@ -77,7 +77,20 @@ public static Guid GetRelativityFieldGuidOfProperty<T>(string propertyName)
 			return fieldAttribute?.FieldGuid ?? new Guid();
 		}
 
-		public PropertyInfo GetParentArtifactIdProperty()
+	    public static RdoFieldType GetRelativityFieldTypeOfProperty<T>(string propertyName)
```

Comment:
Consolidate this with the method above it.

On a side note. in the long run, we will probaby want some sort of cache for all operations.

Reply:
What do you suggest here?  Maybe on method that return an object with all the attribute properties?

Reply:
I guess I don't know what you mean by selector?  Like an Enum as parameter?

Reply:
Ok, made the change, not sure if it is what you were going after.  Still need the string of the property name. but used a selector for the custom attribute.

Reply:
Yes; almost anything you're doing with attributes probably has a matching extension or static method already. If you can't figure it out, ping me. (If anything, we need fewer of these methods, not more.)